src/ostree/ot-admin-builtin-status.c

```diff
@@ -30,13 +30,21 @@
 
 static gboolean opt_verify;
 static gboolean opt_skip_signatures;
-
-static GOptionEntry options[]
-    = { { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
-          NULL },
-        { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures,
-          "Print the commit verification status", NULL },
-        { NULL } };
+static gboolean opt_query_booted;
+
+static GOptionEntry options[] = {
+  { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
+    NULL },
+  { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures, "Skip signatures in output",
+    NULL },
+  { "query-booted", 'Q', 0, G_OPTION_ARG_NONE, &opt_query_booted,
+    "Output the string \"default\" if the default deployment is the booted one, \"not-default\" if "
+    "we are booted in a non-default deployment (e.g. the user interactively chose a different "
+    "entry in  the bootloader menu, or the bootloader rolled back automatically, etc.). If we are "
+    "not in a booted OSTree system, an error is returned.",
```

Comment:
Minor: this is duplicating the manpage. These docstrings are usually more brief. Maybe e.g.

> Output \"default\" if booted into the default deployment, otherwise \"not-default\"

?